runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -160,10 +160,16 @@ getReturnTypeFromMetaData(J9UpcallMetaData *data)
 {
 	J9JavaVM *vm = data->vm;
 	J9VMThread *currentThread = currentVMThread(vm);
-	j9object_t methodType = J9VMOPENJ9INTERNALFOREIGNABIUPCALLMHMETADATA_CALLEETYPE(currentThread,
-			J9_JNI_UNWRAP_REFERENCE(data->mhMetaData));
-	J9Class *retTypeClass = J9VM_J9CLASS_FROM_HEAPCLASS(currentThread,
-			J9VMJAVALANGINVOKEMETHODTYPE_RTYPE(currentThread, methodType));
+	j9object_t methodType = J9VMJAVALANGINVOKEMETHODHANDLE_TYPE(
+									currentThread,
+									J9VMOPENJ9INTERNALFOREIGNABIUPCALLMHMETADATA_CALLEEMH(
+													currentThread,
+													J9_JNI_UNWRAP_REFERENCE(data->mhMetaData)));
+	J9Class *retTypeClass = J9VM_J9CLASS_FROM_HEAPCLASS(
+									currentThread,
+									J9VMJAVALANGINVOKEMETHODTYPE_RTYPE(
+													currentThread,
+													methodType));
```

Comment:
Please use just two tabs per level. Lines 170-172 don't need to be split.

Reply:
Fixed.